src/activities/content/ContentWorkingCopyEntity.js

```diff
@@ -0,0 +1,51 @@
+import { Entity } from '../../es6/Entity';
+import { Actions } from '../../hypermedia-constants';
+import { performSirenAction } from '../../es6/SirenAction';
+
+export class ContentWorkingCopyEntity extends Entity {
```

Comment:
I saw some discussion by @CodeBaboon, @quinnbudan and @ianlaidlaw (sorry if I missed anyone) about extracting the working copy stuff into its own entity, which can then be extended. Here is a small stab at it. I figure weblinks will eventually extend this class (or something like it) as well.

Reply:
This is great!

Reply:
love it!